### DISCUSSION TITLE
[M-Z] (2024/25) - sezione 0.9 es 4.41 (11 novembre)

### DISCUSSION THREAD

**Discussion by AglaiaNorza** [sapienzastudentsnetwork/calcolo-delle-probabilita#318] [Esercizi M-Z]
![es41](https://github.com/user-attachments/assets/2389d62a-0fc9-467b-829c-870f6a99932c)

**Comment by AglaiaNorza** [comment]
![es](https://github.com/user-attachments/assets/43426898-6d7d-4107-8227-cdd0324793f1)

**Comment by marcoseverini** [comment]
![immagine](https://github.com/user-attachments/assets/eddaa404-74a4-46a4-9a6a-0752746cbff7)
